capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -165,8 +165,15 @@ function c4m_restful_quick_post_c4m_location_field_entity_property_info(&$info,
 function c4m_restful_quick_post_preprocess_html(&$vars) {
   $item = menu_get_item();
 
-  if (!menu_get_object() && isset($item['map']) && $item['map'][0] != 'node') {
-    // Not a node page and not a node/add(edit) page.
+  $allowed_paths = array(
+    'node/add',
+    'node/%/edit',
+    'node',
```

Comment:
Pages with menu item path 'node' are a groups' dashboards. So we need app on this page. For example, on some discussion page application is not loading.
![screenshot from 2015-01-20 15 55 17](https://cloud.githubusercontent.com/assets/8306902/5818262/df3a39fa-a0bc-11e4-8dc0-fa8e66dd28fc.png)